_grab_ui.py

```diff
@@ -495,13 +516,15 @@ def perform_action(self, action: Tuple[ActionName, ActionArgs]) -> None:
     def quit(self, *args: Any) -> None:
         self.quit_loop(1)
 
-    region_types = {'stream': StreamRegion,
+    region_types = {'stream':   StreamRegion,
+                    'line':     LineRegion,
                     'columnar': ColumnarRegion
                    }  # type: Dict[RegionTypeStr, Type[Region]]
 
     mode_types = {'normal': NoRegion,
                   'visual': StreamRegion,
-                  'block': ColumnarRegion,
+                  'line':   LineRegion,
+                  'block':  ColumnarRegion,
```

Comment:
In Python, we do not align dictionaries in two columns. When you do, you have to realign each line when you add a longer key, and that causes spurious diffs.

Here, you add one entry to each of two dictionaries. I want to see one line of diff for each.